### ISSUE TITLE
Doest load static in development

### ISSUE THREAD

**Issue by henryfool91** [tolomea/django-data-browser#45]
Hello! I`m trying to install your package to localhost, but it seem that it doesnt work cause of some staticfiles issues, could you help to get over it? It works fine on production (docker) server btw! May be there is some issues with my staticfiles django settings, or may be there is something else that i`m overlooked? Thanks in advance! 

```
Django 4.0.1
settings.py

Debug = True
STATIC_URL = 'static/'
STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
```
![Снимок экрана 2022-03-15 215911](https://user-images.githubusercontent.com/82047953/158452209-c25a5363-3045-44ea-a513-8e28eae10caf.png)

**Comment by tolomea** [comment]
Sorry I haven't replied to this yet. I feel like I've seen this before but I can't think where.
One thing I can say is it doesn't use the static file settings, because it wants to work regardless of your environment it does it's own static file serving.

**Comment by tolomea** [comment]
Does everything look normal on the network tab? you aren't getting 404's or anything?

**Comment by tolomea** [comment]
Not Dj4, my local works fine with Dj4

**Comment by henryfool91** [comment]
Yep, may be! As for now, i am more than ok with my temporary fix cause anyway i have separated settings for prod and local development, just thougth it could be userfull for somebody with same problem.
Closing issu
And, btw, thanks for great repository!

**Comment by tolomea** [comment]
As it happens the new starter at my day job is having exactly the same issue
